domains/list.go

```diff
@@ -17,7 +26,7 @@ func List(app string) error {
 	}
 
 	t := tablewriter.NewWriter(os.Stdout)
-	t.SetHeader([]string{"Domain", "SSL"})
+	t.SetHeader([]string{"Domain", "TLS/SSL"})
```

Comment:
You could not resist? :stuck_out_tongue:

Reply:
:laughing: